doc/cpp/morphology.rst

```diff
@@ -200,6 +233,12 @@ by two stitches:
 
    cable_cell cell(stitched.morphology(), dec, stitched.labels());
 
+Debug Ouput
+-----------
+
+Tree representations of :cpp:type:`segment_tree` and :cpp:type:`morphology` can be obtained
+by including ``arborio/debug.hpp`` which contains a series of :cpp:func:`show` functions
+that return ASCII renderings of the given object.
 
```

Comment:
you could add the example output from the PR description here

Reply:
Good idea